bftengine/src/bftengine/messages/ClientRequestMsg.cpp

```diff
@@ -150,6 +150,12 @@ void ClientRequestMsg::validateImp(const ReplicasInfo& repInfo) const {
       }
     }
   }
+  if ((isClientTransactionSigningEnabled == false) && (header->flags & RECONFIG_FLAG)) {
```

Comment:
`isClientTransactionSigningEnabled == false` is not correct here, because in real deployment the operator is not part of any client group, so its messages cannot be verified here

Reply:
We are skipping signature verification here. Reason for adding this check here is cre requests are always signed. If tx signing is disabled, we need to skip verification here and it is verified by reconfiguration handler